cart/templates/cart.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'base.html' %} {% load static from staticfiles %} {% load bootstrap_tags %} {% block page_heading %}Cart{% endblock %} {% block content %}
+<div class="row row-flex">
+    {% for item in cart_items %}
+    <div class="col-xs-10 col-xs-offset-1 col-sm-6 col-sm-offset-0 col-md-4 display panel panel-default">
+        <div class="cart-img" style="background-image: url('{{ MEDIA_URL }}{{ item.product_variant.product.image }}')"></div>
+        <div class="caption">
+            <h5>{{ item.product_variant.product.name }}</h5>
+            <p>({{ item.product_variant.size }})</p>
+            <p>€{{ item.product_variant.price }} <small>(per copy)</small></p>
+            <form class="form-row" method="POST" action="{% url 'edit_cart' item.id %}">
+                {% csrf_token %}
+                <div class="input-group mb-2">
+                    <div class="input-group-prepend">
+                        <div class="input-group-text">Qty: {{ item.quantity }}</div>
+                    </div>
+                    <input class="cart-input" type="number" name="quantity" min="0" max="999" class="form-control" placeholder="Edit Qty" />
+                    <button type="submit" class="btn btn-primary"><i class="fa fa-pencil-square-o" aria-hidden="true"></i></button>
+                </div>
+            </form>
+        </div>
+    </div>
+    {% endfor %}
+</div>
+
+<div class="center-text">
+    {% if total != 0 %}
```

Comment:
do you need the `<div class="center-text">` if this condition fails? i think the `if statement` should come before the div

Reply:
Resolved